builder/vsphere/clone/step_customize.go

```diff
@@ -49,6 +51,39 @@ type LinuxOptions struct {
 	Timezone string `mapstructure:"time_zone"`
 }
 
+type WindowsOptions struct {
+	// CustomizationGuiRunOnce
+	// A list of commands to run at first user logon, after guest customization.
+	RunOnceCommandList *[]string `mapstructure:"run_once_command_list"`
+	// CustomizationGuiUnattended
+	// Specifies whether or not the VM automatically logs on as Administrator.
+	AutoLogon *bool `mapstructure:"auto_logon"`
+	// Specifies how many times the VM should auto-logon the Administrator account when auto_logon is true. Default 1
+	AutoLogonCount *int32 `mapstructure:"auto_logon_count"`
+	// The new administrator password for this virtual machine.
+	AdminPassword *string `mapstructure:"admin_password"`
+	// The new time zone for the virtual machine. This is a sysprep-dictated timezone code. Default 85 (GMT)
+	TimeZone *int32 `mapstructure:"time_zone"`
+	// CustomizationIdentification
+	// The user account of the domain administrator used to join this virtual machine to the domain.
+	DomainAdminUser string `mapstructure:"domain_admin_user"`
+	// The password of the domain administrator used to join this virtual machine to the domain.
+	DomainAdminPassword string `mapstructure:"domain_admin_password"`
+	// The domain that the virtual machine should join.
+	JoinDomain string `mapstructure:"join_domain"`
```

Comment:
I don't think it makes good sense to include the ability to join the machine image to a domain per comments in #34.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

Reply:
OK.